CHANGELOG.md

```diff
@@ -2,20 +2,20 @@
 
 ## Unversioned
 
-- Major: Added live emote updates for BTTV (#4147)
-- Minor: Added option to highlight your own messages in Highlights page under Users tab. (#3833)
-- Minor: Change the highlight order to prioritize Message highlights over User highlights. (#4303)
-- Minor: Added ability to negate search options by prefixing it with an exclamation mark (e.g. `!badge:mod` to search for messages where the author does not have the moderator badge). (#4207)
+- Major: Added live emote updates for BTTV. (#4147)
+- Minor: Added the option to highlight your own messages in the Highlights page under the Users tab. (#3833)
+- Minor: Changed the highlight order to prioritize Message highlights over User highlights. (#4303)
+- Minor: Added the ability to negate search options by prefixing it with an exclamation mark (e.g. `!badge:mod` to search for messages where the author does not have the moderator badge). (#4207)
 - Minor: Search window input will automatically use currently selected text if present. (#4178)
-- Minor: Cleared up highlight sound settings (#4194)
+- Minor: Cleared up highlight sound settings to reduce confusion. (#4194)
```

Comment:
I don't think adding `to reduce confusion` makes this changelog entry any better.
If we want to clarify, an alternative would be something along the lines of "Group highlight sound columns together and improve wording for the default sound setting.". I'm impartial to this being 1 or 2 changelog entries.